src/dune_rules/melange/melange_rules.ml

```diff
@@ -172,6 +172,15 @@ let build_js ~loc ~dir ~pkg_name ~mode ~module_systems ~output ~obj_dir ~sctx
            ; Dep src
            ]))
 
+let add_deps_to_default_ailases ~dir deps =
+  let open Memo.O in
+  let melange_implicit_alias =
+    Alias.make Melange_stanzas.Emit.implicit_alias ~dir
+  in
+  let dune_default_alias = Alias.make Alias.Name.all ~dir in
```

Comment:
This should be `Alias.all ~dir`

Reply:
fixed in https://github.com/ocaml/dune/pull/7926/commits/e213c26fbc8023418261d5ea67f7ffdf71e7abfc